pyccel/ast/numpy_wrapper.py

```diff
@@ -66,15 +68,15 @@ def get_numpy_max_acceptable_version_file():
                              results   = [Variable(dtype=NativeInteger(), name = 'i', precision = 4)])
 
 # numpy array to c ndarray : function definition in pyccel/stdlib/cwrapper.c
-pyarray_to_c_ndarray = FunctionDef(
-                name      = 'pyarray_to_c_ndarray',
+pyarray_to_ndarray = FunctionDef(
+                name      = 'pyarray_to_ndarray',
                 arguments = [Variable(name = 'a', dtype = PyccelPyArrayObject(), is_pointer = True)],
                 body      = [],
                 results   = [Variable(name = 'array', dtype = NativeGeneric())])
 
 # numpy array check elements : function definition in pyccel/stdlib/cwrapper.c
 pyarray_checker = FunctionDef(
-                name      = 'pyarray_checker',
+                name      = 'pyarray_check',
```

Comment:
I think you forget to change the name of the function too.
you change here only the name attribute.

Reply:
I changed the name attribute to match the name in the c code. I'm not sure why this change was made so I didn't see the need to propagate it through the rest of the code. I can though if you think it's clearer

Reply:
Fixed 330e8b6b